.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Java CI with Maven
+on: push
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: maven
+      - name: build with Maven
+        run: mvn -B package --file pom.xml
+  test:
+    needs: [build]
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: use Telegram Token
+        run: echo "Using token ${{ secrets.BOT_TOKEN }}"
+        env:
+          BOT_TOKEN: ${{ secrets.BOT_TOKEN }}
+      - name: Test
```

Comment:
You must not to use Telegram bot tokens in tests. Mock external dependencies if needed.